hydra-node/test/Hydra/LoggingSpec.hs

```diff
@@ -21,10 +21,9 @@ spec = do
 
     captured `shouldContain` "{\"foo\":42}"
 
-  prop "HydraLog" $
-    property $
-      withMaxSuccess 1 $
-        conjoin
-          [ prop_validateJSONSchema @(Envelope (HydraLog Tx ())) "logs.json" id
-          , prop_specIsComplete @(HydraLog Tx ()) "logs.json" (key "properties" . key "message")
-          ]
+  modifyMaxSuccess (const 10) $
+    prop "Validates logs.yaml schema" $
+      prop_validateJSONSchema @(Envelope (HydraLog Tx ())) "logs.json" id
+  modifyMaxSuccess (const 10) $
+    prop "Schema covers all defined log entries" $
+      prop_specIsComplete @(HydraLog Tx ()) "logs.json" (key "properties" . key "message")
```

Comment:
I don't understand the rationale of this commit. The commit message also does not enlighten me. Why is this "more explicit"? I vaguely recall there was a reason for `withMaxSuccess` .. maybe because the external tool was so slow and we do generate many entries instead? (It's [not so many anymore](https://github.com/input-output-hk/hydra/blob/9962f34c5b06d29aa8463121453ff9a9929372d5/hydra-node/test/Hydra/JSONSchema.hs#L50) as it seems).

Reply:
I had a failure and was confused, could not tell which of the 2 sub-properties actually failed, and using `conjoin` seems to obscure things more than help. So I figured it would be clearer to have separate, named properties.
As for the change in the number of successes, it seems those tests are not so slow anymore but I am fine reverting to 1

Reply:
Makes sense to me and would have approved with this rationale. Anyhow. It's a smaller change now :shrug: